### DISCUSSION TITLE
Error installing apps

### DISCUSSION THREAD

**Discussion by sandramgomezr** [Homebrew/discussions#1786] [Getting started]
I got this error trying to install maven or openjdk:

Error: Failed to download resource "zip_bottle_manifest"
Download failed: https://ghcr.io/v2/linuxbrew/core/zip/manifests/3.0-2

**Comment by SMillerDev** [comment]
Try running `brew update` and if that doesn't fix it, post the output of `brew config` and `brew doctor`
